coalib/output/ConsoleInteraction.py

```diff
@@ -199,6 +199,112 @@ def acquire_actions_and_apply(console_printer,
             break
 
 
+def get_context(console_printer, file_dict,
+                sourcerange):  # pragma: no cover
+    """
+    Gets context for the affected lines and also the line numbers
+    of the context
+
+    :param file_dict:       A dictionary containing all files as values with
+                            filenames as key.
+    :param sourcerange:     The SourceRange object referring to the related
+                            lines to print.
+
+    :yield:                 Affected file's contextual information
+
+    """
+
+    related_lines = list()
+    related_lines_nr = list()
```

Comment:
Please explain what is `related_lines_nr`. Possibly change name of variable so it becomes self evident.